### ISSUE TITLE
[BUG] Mystical Agriculture Augment(s) not working

### ISSUE THREAD

**Issue by Go0gAm0o** [TeamAOF/All-of-Fabric-3#447]
**Is this a modpack or mod issue?**
(if something is screwy using a certain mod, then send the bug to the mod dev first.)
Not sure

**PLEASE MENTION IF YOU ADDED ANY MODS.. ESPECIALLY SODIUM, IF YOU DON'T, IT MAKES US VERY GRUMPY**

**Modpack Version**
(any bugs using a non-up-to-date version will be asked to replicate with the most up-to-date release)
2.8.5

**Describe the bug**

I crafted the night vision augment and applied it to the Supremium helmet, but it doesnt give the night vision effect. I also checked to see if it was a simply a keybind issue but mystical agriculture isn't even in the list to give a keybind.

**To Reproduce**
1. Craft the Night Vision Augment
2. Apply augment to Supremium Helmet
3. Put helmet on and augment does not give night vision effect

**Expected behavior**
Expected to get night vision effect

**Screenshots**

**Logs**
(you can use https://paste.alloffabric.com)

**Additional context**

**Comment by Mendanbar** [comment]
I've had the same problem with Fire Resistance and Water Breathing.  The flight augment works though.

**Comment by Go0gAm0o** [comment]
I've tested some more out as well myself and agree with you on the Fire Resistance and Water Breathing augments not working. Haven't got to make the flight augment just yet.

**Comment by i-g0r** [comment]
I think has been fixed in [1.16.5-1.1.7](https://www.curseforge.com/minecraft/mc-mods/mystical-agriculture-refabricated/files/3362276) release

**Comment by Mendanbar** [comment]
Yep confirmed here too.  Updated to 1.1.7 and now water breathing and fire resistance are working.
